shell/platform/darwin/macos/framework/Source/FlutterView.h

```diff
@@ -18,7 +18,7 @@
  * backward compatibility, single-view APIs will always operate the view with
  * this ID. Also, the first view assigned to the engine will also have this ID.
  */
-constexpr uint64_t kFlutterDefaultViewId = 0;
+constexpr uint64_t kFlutterDefaultViewId = 1;
```

Comment:
Just so this comment isn't lost: https://github.com/flutter/engine/pull/38855#discussion_r1073934050

Consider double-checking with @goderbauer that the framework does not have any expectations that the default view has an ID of `0`.

Reply:
The framework makes no assumption about the value of the id. Why does this have to change from 0 to 1, though?